
  sed in debian/rules deletes "__unused" everywhere,
  fix consequences now

diff -ur src/usr.bin/yacc/skeleton.c src/usr.bin/yacc/skeleton.c
--- src/usr.bin/yacc/skeleton.c
+++ src/usr.bin/yacc/skeleton.c
@@ -60,9 +60,6 @@
 {
     "#include <stdlib.h>",
     "#ifndef lint",
-    "#ifdef ",
-    " ",
-    "#endif",
     "static char const ",
     "yyrcsid[] = \"$FreeBSD: src/usr.bin/yacc/skeleton.c,v 1.37 2003/02/12 18:03:55 davidc Exp $\";",
     "#endif",
diff -ur src/usr.bin/lex/flex.skl src/usr.bin/lex/flex.skl
--- src/usr.bin/lex/flex.skl
+++ src/usr.bin/lex/flex.skl
@@ -8,7 +8,6 @@
 #if defined(__FreeBSD__)
 #include <sys/cdefs.h>
 #else
-#define 
 #endif
 
 #define FLEX_SCANNER
